tests/scene/test_arraymesh.h

```diff
@@ -88,6 +88,25 @@ TEST_CASE("[SceneTree][ArrayMesh] Adding and modifying blendshapes.") {
 		CHECK(mesh->get_blend_shape_count() == 0);
 	}
 
+	SUBCASE("Adding blend shapes once all surfaces have been removed is allowed") {
+		Ref<CylinderMesh> cylinder = memnew(CylinderMesh);
```

Comment:
I wrote it like this because it's the local coding style used in this file. Can I change the whole file with your suggestions or would this be too much for this PR?

Reply:
Just change your own code for this

Reply:
I think the local coding style should be respected. It's way worse for everyone to have two arbitrary code styles in the document. Further, its unfair to ask a contributor to make an arbitrary style change after they have been waiting so long for an approval...

Reply:
I don’t mind updating the whole file, it’s a 5 min work. Would you prefer I do it in this PR or a separate one?

Reply:
A separate one! I'd like to get this PR merged as soon as possible so we can include it in 4.4

Reply:
Fair enough but this isn't an arbitrary style change but a well established one, but can absolutely be done in a followup for these parts